devel/config-single.yaml

```diff
@@ -35,10 +35,12 @@ admin:
   enabled: true
   address: 0.0.0.0
   port:    2827
+  username: admin
+  passsword: s3cr3t
 
 # The node will self-ban but this section needs at least one entry
 network:
-  - http://127.0.0.1:2826/
+  - https://127.0.0.1:2826/
```

Comment:
Do we need this change ?

Reply:
Without it will keep logging error as it tries to connect and before it is banned it fails to connect without ssl.

Reply:
Alright, I think that's another issue, actually.
The current state of things mean that we can't have an administrative interface enabled AND an HTTP interface, because the http code automatically picks up certificates and just switch to https. Let's talk about this IRL.